translations/en-us.yaml

```diff
@@ -2237,6 +2238,35 @@ formGlobalRoles:
       label: Use Node Templates
       detail: Allows the user to deploy new Nodes using any existing Node Templates.
 
+formScopedRoles:
+  title: '{type} Permissions'
+  description: TBD.
+  mode:
+    admin:
+      label: '{type} owner'
+      detail: Administrators have full control over the entire installation and all resources in all clusters.
+    user:
+      label: '{type} member'
+      detail: Standard Users can create new clusters and manage clusters and projects they have been granted access to.
+    custom:
+      label: Custom
+      detail: Choose individual permissions for the user.
+  role:
+    cluster-owner:
```

Comment:
These need to depend on the resource being a cluster or project ($TYPE)

```
mode:
  admin: // this should probably be renamed owner
    label: Owner
    detail: Owners have full control over the $TYPE and all resources inside it.
  user:  // ... member
    label: Member
    detail: Members can manage the resources inside the $TYPE but not change the $TYPE itself.
  custom:
    label: Custom
    label: Choose individual roles for this user

role:
   read-only:
    label: Read-Only
    detail: Allows seeing everything, but not changing anything.

  // Drop, these won't be shown because the radio for admin/user picks them
  cluster-owner: drop
  cluster-member:  drop
  project-owner: drop
  project-member: drop

  // There will be more fine-grained roles from craig coming...
```